tests/installation/boot_into_snapshot.pm

```diff
@@ -23,40 +21,19 @@ use base "opensusebasetest";
 
 sub run {
     my ($self) = @_;
-    assert_screen 'linux-login', 200;
-    select_console 'root-console';
-    # 1)
-    assert_script_run('touch /etc/NOWRITE;test ! -f /etc/NOWRITE');
-    # 1b) just debugging infos
-    assert_script_run("snapper --iso list");
-    assert_script_run("cat /etc/os-release");
-    if (get_var("UPGRADE")) {
-        # if we made a migration, the version should be for example opensuse before migr. 42.1 > 42.2
-        # extract number of version id: example SlES 12.2 -> 12.2. for opensuse also ok
-        my $OS_VERSION     = script_output("grep VERSION_ID /etc/os-release | cut -c13- | head -c -2");
-        my $OLD_OS_VERSION = script_output("grep VERSION_ID /.snapshots/2/snapshot/etc/os-release | cut -c13- | head -c -2");
-        # grub_bug bug:956046. menu entry not stable. we could boot in wrong menu-entry.
-        if ($OS_VERSION eq $OLD_OS_VERSION) {
-            die "OS_VERSION after Rollback matches OS_VERSION before Rollback";
-        }
+    if (check_var('DESKTOP', 'textmode')) {
+        assert_screen('linux-login', 200);
+    }
+    else {
+        assert_screen('generic-desktop', 200);
     }
-    assert_script_run('snapper rollback');
-    assert_script_run('snapper --iso list');
-    assert_script_run('snapper --iso list | grep \'number.*first root filesystem.*|\s*$\'',
-        fail_message => 'first root filesystem should be marked for cleanup as well');
-    assert_script_run('snapper --iso list | grep \'number.*after installation\'.*important=yes');
-    assert_script_run('snapper --iso list | grep \'number.*rollback\'.*important=yes',
-        fail_message => 'rollback backup should be marked for cleanup fate#321773');
-    assert_script_run('snapper --iso list | tail -n 1 | grep \'|\s*| [^|]\+ |\s*$\'', fail_message => 'last snapshot should not be cleaned up but is not-important');
+    select_console 'root-console';
+    assert_script_run('touch /etc/NOWRITE;test ! -f /etc/NOWRITE') if (get_var('BOOT_INTO_SNAPSHOT'));
     script_run("systemctl reboot", 0);
     reset_consoles;
     $self->wait_boot;
```

Comment:
This is wrong. Let's talk about `tests/installation/boot_into_snapshot.pm` when the other changes are done.

Keep in mind the schedules stated in tasks: https://progress.opensuse.org/issues/12964#Tasks